package.tsconfig.json

```diff
@@ -15,8 +15,11 @@
     "paths": {
       "@leafygreen-ui/*": ["packages/*/src"]
     },
-    "emitDeclarationOnly": true,
+    "sourceMap": true,
+    "declarationMap": true,
+    "emitDeclarationOnly": false,
+    "importHelpers": true,
     "composite": true
   },
-  "exclude": ["*.story.*", "*.spec.*", "dist"]
+  "exclude": ["**/*/*.spec.*", "**/*/*.story.*"]
```

Comment:
Do we not want to exclude compiling dist directories?

Reply:
It seemed to not make a difference afaict. I can iterate on it more, but didn't seem to make a difference.